ios/FluentUI/Vnext/List/ListTokens.swift

```diff
@@ -38,96 +38,63 @@ import SwiftUI
 
 // MARK: ListCell Tokens
 
-class MSFCellBaseTokens: MSFTokensBase, ObservableObject {
-    @Published public var borderColor: UIColor!
-    @Published public var disclosureIconForegroundColor: UIColor!
-    @Published public var labelColor: UIColor!
-    @Published public var leadingViewColor: UIColor!
-    @Published public var sublabelColor: UIColor!
-    @Published public var trailingItemForegroundColor: UIColor!
-
-    @Published public var backgroundColor: UIColor!
-    @Published public var highlightedBackgroundColor: UIColor!
-
-    @Published public var cellHeightOneLine: CGFloat!
-    @Published public var cellHeightTwoLines: CGFloat!
-    @Published public var cellHeightThreeLines: CGFloat!
-    @Published public var disclosureInterspace: CGFloat!
-    @Published public var disclosureSize: CGFloat!
-    @Published public var horizontalCellPadding: CGFloat!
-    @Published public var iconInterspace: CGFloat!
-    @Published public var labelAccessoryInterspace: CGFloat!
-    @Published public var labelAccessorySize: CGFloat!
-    @Published public var leadingViewSize: CGFloat!
-    @Published public var leadingViewAreaSize: CGFloat!
-    @Published public var sublabelAccessorySize: CGFloat!
-    @Published public var trailingItemSize: CGFloat!
-    @Published public var verticalCellPadding: CGFloat!
-
-    @Published public var footnoteFont: UIFont!
-    @Published public var sublabelFont: UIFont!
-    @Published public var labelFont: UIFont!
-
-    var cellLeadingViewSize: MSFListCellLeadingViewSize! {
-        didSet {
-            if oldValue != cellLeadingViewSize {
-                updateForCurrentTheme()
-            }
-        }
-    }
-}
+public class MSFCellBaseTokens: ControlTokens {
```

Comment:
Two points:
1. This class should be `open` so that clients outside `FluentUI` will be able to subclass it.
2. `Tokens` classes are one of the ones we want to drop the `MSF` prefix from. So this should just be `CellBaseTokens`.